scripts/assistant/start.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+# Copyright OpenSearch Contributors
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+
+. scripts/assistant/utils.sh
+. scripts/assistant/add_model.sh
+
+function usage() {
+    echo ""
+    echo "This script is used to run OpenSearch Assistant"
+    echo "--------------------------------------------------------------------------"
+    echo "Usage: $0 [args]"
+    echo ""
+    echo "Optional arguments:"
+    echo -e "-b BIND_ADDRESS\t, defaults to localhost | 127.0.0.1, can be changed to any IP or domain name for the cluster location."
+    echo -e "-p BIND_PORT\t, defaults to 5601 depends on OpenSearch or Dashboards, can be changed to any port for the cluster location."
+    echo -e "-c CREDENTIAL\t(password), defaults to admin"
+    echo -e "-h\tPrint this message."
+    echo "--------------------------------------------------------------------------"
+}
+
+while getopts ":h:b:p:c:" arg; do
+    case $arg in
+        h)
+            usage
+            exit 1
+            ;;
+        b)
+            BIND_ADDRESS=$OPTARG
+            ;;
+        p)
+            BIND_PORT=$OPTARG
+            ;;
+        c)
+            CREDENTIAL=$OPTARG
+            ;;
+        :)
+            echo "-${OPTARG} requires an argument"
+            usage
+            exit 1
+            ;;
+        ?)
+            echo "Invalid option: -${OPTARG}"
+            exit 1
+            ;;
+    esac
+done
+
+[ -z "$BIND_ADDRESS" ] && BIND_ADDRESS="localhost"
+[ -z "$BIND_PORT" ] && BIND_PORT="9200"
+[ -z "$CREDENTIAL" ] && CREDENTIAL="admin"
+[ -z "$REGION" ] && REGION="us-west-2"
+
+PARENT_PID_LIST=()
+
+PACKAGE_VERSION=$(yarn --silent pkg-version)
+
+# define assistant path
+CWD=$(pwd)
+SNAPSHOT_DIR="$CWD/.opensearch"
+LOGS_DIR="$SNAPSHOT_DIR/$PACKAGE_VERSION/logs"
+
+# Main function
+function execute() {
+  export initialAdminPassword=$CREDENTIAL
+  CLUSTER_SETTINGS="snapshot --assistant --security"
+  CLUSTER_SETTINGS+=" -E plugins.ml_commons.only_run_on_ml_node=true"
+  
+  run_opensearch || clean
+  check_opensearch_status
+  echo "Attempting to add models..."
+  echo "(Ensure your environment is exporting your credentials)"
+  (add_model > $LOGS_DIR/add_model.log 2>&1 || clean) & 
+
+  export OPENSEARCH_USERNAME=kibanaserver
+  export OPENSEARCH_PASSWORD=kibanaserver
```

Comment:
Yeah I wish we can change this username for this user since I think it's confusing. This is an [internal user](https://github.com/opensearch-project/security/blob/481b373121a05cac0ce45b05a913c68912d161cd/config/internal_users.yml#L30). Which I feel like it's not accurate by the description linked as it doesn't feel like a demo user since it has super privileges. It also seems to be able to ignore attempts to remove privileges. 

This script utilizes snapshots and anytime that is ran the snapshot will get wiped out clean and will have this super user. Which really is just used for the health check. This part is kind of more like an "easy install". What this does is call to OSD scripts to start up the cluster with security plugin and assistant required plugins. Run all the models stuff and then start up OSD. However, if the user would like to change some stuff then can do that stuff a little bit manually with the following:

In a window run the OpenSearch snapshot:
```
yarn opensearch snapshot --assistant --security  -E plugins.ml_commons.only_run_on_ml_node=true
```

run the curl requests.

and then in another window starting up a process of OpenSearch Dashboards

```
yarn start --security
```

With that said, I can still modify if a hard blocker. Lemme know!